packages/parany/parany.13.0.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: "Francois Berenger"
+license: "LGPL-2.0-or-later"
+homepage: "https://github.com/UnixJunkie/parany"
+bug-reports: "https://github.com/UnixJunkie/parany/issues"
+dev-repo: "git+https://github.com/UnixJunkie/parany.git"
+depends: [
+  "domainslib"
```

Comment:
There's a lower-bound failure which suggests that there should be a constraint here:

```suggestion
  "domainslib" {>= "0.5.0"}
```